samples/word/99-preview-apis/manage-fields.yaml

```diff
@@ -0,0 +1,347 @@
+id: word-manage-fields
```

Comment:
if this preview script is just for Fileld.showCodes then I think there is no need to keep this one for several reasons:
1. showCodes is not that important, we do not necessarily add code samples for each of the APIs we are providing.
2. there are big limitations on wac to implement this, so I am afraid so far there is no plan to further ship it in the future, so we don't want to promote it.

Reply:
removed